src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -189,8 +189,6 @@ private void OnErrorDataReceived(object sender, DataReceivedEventArgs e)
 
         private void TerminateRunningProcess()
         {
-            logger.LogDebug(Resources.INFO_TerminatingServer);
```

Comment:
Since we now call this when solution closes & settings change, this was getting a bit spammy as maybe we never analyzed a JS file and therefore there was no process running.
So I moved it to the block where there definitely is a process and logged the process id as well.